tests/process.issue.test.ts

```diff
@@ -1 +1,44 @@
 
+import { parseGitHubUrl } from "../src/start";
+import { IssueActivity } from "../src/issue-activity";
+import { Processor } from "../src/parser/processor";
+import { UserExtractorModule } from "../src/parser/user-extractor-module";
+import { server } from "./__mocks__/node";
+import { DataPurgeModule } from "../src/parser/data-purge-module";
+import userCommentResults from "./__mocks__/results/user-comment-results.json";
+import dataPurgeResults from "./__mocks__/results/data-purge-result.json";
+
+const issueUrl = process.env.TEST_ISSUE_URL || "https://github.com/ubiquibot/comment-incentives/issues/22";
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe("Modules tests", () => {
+  // parse the issue url
+  const issue = parseGitHubUrl(issueUrl);
+  const activity = new IssueActivity(issue);
+
+  beforeAll(async () => {
+    await activity.init();
+    await activity.init();
```

Comment:
```suggestion
    await activity.init();
```
Initialize only one time.

Reply:
resolved.